app/lib/hosting_environment.rb

```diff
@@ -2,4 +2,8 @@ module HostingEnvironment
   def self.production?
     Rails.env.production?
   end
+
+  def self.loadtest?
+    Rails.env.loadtest?
```

Comment:
I don't think we have a `loadtest.rb` in `environments`?

Reply:
We have one on Find's load-test branch!

Reply:
Aha!